text.html

```diff
@@ -74,136 +88,198 @@ <h2 class="title">Tonic</h2>
-                        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <p class="description">A smart assistant to make driving more safe, efficient, and fun by
+                            unlocking your most expensive computer: your car.</p>
                         <ul class="languages">
                             <li class="stack">html</li>
+                            <li class="stack">Ruby on rails</li>
                             <li class="stack">css</li>
                             <li class="stack">javascript</li>
                         </ul>
-                        <button class="btn">See Project </button>  
-                    </div> 
+                        <button class="btn">See Project </button>
+                    </div>
                 </article>
-            </section>  
-            <section class="about">
-                <div></div>
-                <div class="greet">
-                    <h1>About Myself</h1>
-                </div> 
-                <div class="presentation">
-                    <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-                </div>
-                <div class="connexion">
-                    <p><a href="#">LET'S CONNECT</a><p>
-                </div>
-                <div class="social">
-                    <ul class="icons">
-                        <li>
-                            <a href="#"><svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <path d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z" fill="#505F79"/>
-                                </svg></a>
-                        </li>
-                        <li>
-                            <a href="#"><img src="Linkedinicon.svg"/></a>
-                        </li>
+            </section>
+            <section class="about" id="About">
+                <section class="about1">
+                    <div></div>
+                    <div class="greet">
+                        <h1>About Myself</h1>
+                    </div>
```

Comment:
Kindly increase the font size of this heading to match the Figma design, also please note that there should only be one `h1` tag per page so please change this to `h2,h3... etc.` to follow the best practices 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/192988991-c97f6899-a884-466b-b403-80f9369faba6.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/192989055-535d657b-4e84-4fd2-bffd-e2a4f43798c9.png)